atomos/atomic.py

```diff
@@ -172,7 +177,7 @@ def __init__(self, value=long(0)):
 
     def __setattr__(self, name, value):
         # Ensure the `_value` attribute is always a long.
-        if name == '_value' and not isinstance(value, types.LongType):
+        if name == '_value' and not isinstance(value, long):
```

Comment:
Python 3 don't have this type any more. See http://www.diveintopython3.net/porting-code-to-python-3-with-2to3.html#long

Not sure but using the primitive type can do the job for both Python versions.